26.10.22/string/js/hw3_myskovets.js

```diff
@@ -0,0 +1,10 @@
+var button = document.querySelector('button');
+
+button.onclick = function() {
+  var name = prompt('First part');
+  var name2 = prompt('Second part');
+  var name3 = prompt('Thirt part');
```

Comment:
And use human-readable variable names and massages, pls. It could decrement your mark.